File: DemoMMOUnity/Assets/Game/Hoftix/Script/GenerateScripts/UIModules/OperationalInterfacePanelView.cs
﻿using ZJYFrameWork.UISerializable.UIInitView;
using UnityEngine;
using UnityEngine.UI;

namespace ZJYFrameWork.UISerializable
{
    public class OperationalInterfacePanelView:UIViewInterface
    {
        public UnityEngine.UI.Button ReturnButton=null;
		public UnityEngine.GameObject MapGuideObj=null;
		


        public void Init(UIView _view)
        {
            ReturnButton=_view.GetObjType<UnityEngine.UI.Button>("ReturnButton");
			MapGuideObj=_view.GetObjType<UnityEngine.GameObject>("MapGuideObj");
			
        }
    }
}

File: UnityMMOClient/Lua/application/app/ui/generateScripts/UIModules/OperationalInterfacePanelView.lua
---@class OperationalInterfacePanelView
OperationalInterfacePanelView = class("OperationalInterfacePanelView")
function OperationalInterfacePanelView:Init(view)
	self._UIView = view:GetComponent("UIView")
	self.ReturnButton = self._UIView:GetObjTypeStr("ReturnButton") or UnityEngine.UI.Button
	self.MapGuideObj = self._UIView:GetObjTypeStr("MapGuideObj") or UnityEngine.GameObject
	self.Grid = self._UIView:GetObjTypeStr("Grid") or ZJYFrameWork.Module.UICommon.AdaptiveReuseGrid
end

